lib/fastly/client.rb

```diff
@@ -13,6 +13,7 @@ def initialize(opts)
       @user     = opts.fetch(:user, nil)
       @password = opts.fetch(:password, nil)
       @customer = opts.fetch(:customer, nil)
+      @oldpurge = opts.fetch(:use_old_purge_method, false)  
```

Comment:
Is there any reason that a customer would actually want to use the old method? If not, then I propose that we remove it completely.